instruments/instruments/util_fns.py

```diff
@@ -54,6 +54,41 @@ def assume_units(value, units):
         value = pq.Quantity(value, units)
     return value
 
+
+def convert_temperature(temperature, base):
+    """
+    convert the temperature to the specified base
+    :param temperature: a quantity with units of Kelvin, Celsius, or Fahrenheit
+    :type temperature: `quantities.Quantity`
+    :param base: a temperature unit to convert to
+    :type base: `unitquantity.UnitTemperature`
+    :return: the converted temperature
+    :rtype: `quantities.Quantity`
+    """
+    # quantities reports equivalence between degC and degK, so a string comparison is needed
```

Comment:
Also, this is not the first time we've had a disagreement with them over how units should be handled, see https://github.com/Galvant/InstrumentKit/blob/dev/instruments/instruments/units.py where @cgranade added some additional units to our implementation.